doc/conf.py

```diff
@@ -24,6 +24,13 @@
         "sphinx_copybutton",
         ]
 
+autodoc_typehints = "description"
```

Comment:
Is this ok? It basically moves the types from the function signature to the arguments ([see docs](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints))
* [with 'description'](https://i.imgur.com/NKruh7b.png)
* [with 'signature'](https://i.imgur.com/7Mdly8w.png) (this should be the default)

Reply:
TIL! I actually like this a lot. The way it handles missing docstrings isn't crazy either: it just creates an entry. In fact, I think we should use this across all the packages. I wonder if it's worth it to create some mechanism to configure all the docs centrally, to avoid having to do stupid "PR campaigns" for stuff like this.

(And thanks for pointing it out, I had missed it initially.)